src/content/documentation/publish/add-on-policies-preview-2025-08.md

```diff
@@ -0,0 +1,269 @@
+---
+layout: sidebar
+title: Add-on Policies (Aug 4, 2025)
+permalink: /documentation/publish/add-on-policies-preview-2025-08/
+topic: Publish
+tags: [add-ons, review, policies]
+contributors:
+  [
+    kewisch,
+    rebloor,
+    wagnerand,
+    atsay,
+    jvillalobos,
+    wbamberg,
+    kmaglione,
+  ]
+last_updated_by: wagnerand
+date: 2025-08-04
```

Comment:
why is this date in the far future? this is the last updated date, not when the policies apply.